utils/noCopy/guard.go

```diff
@@ -0,0 +1,6 @@
+package noCopy
+
+type Locker interface {
+	Lock()
+	Unlock()
+}
```

Comment:
Judging by [this answer](https://stackoverflow.com/questions/52494458/nocopy-minimal-example) it looks like the linter does a dumb check on just the two method names (i.e. `Lock()` and `Unlock()`) so I don't think we need this interface. Just having the `struct` with those two empty methods should be enough :+1:

Reply:
Make sense.